lib/tasks/heroku.rake

```diff
@@ -2,41 +2,56 @@
 
 require 'fileutils'
 require 'ipaddr'
+require 'dotenv/tasks'
 
 require_relative '../gem_dandy/github'
 
 namespace :heroku do
-  REPOS_TO_UPDATE = Hash[ENV['REPOS_TO_UPDATE'].split(',').map { |s| s.split(':') }]
-  GITHUB_USER = GemDandy::Github.client.user.login
   SSH_DIR = File.join(ENV['HOME'], '.ssh').to_s
 
   desc 'Write the SSH_PRIVATE_KEY on the remote server'
   task :write_private_key do
-    abort unless ENV['SSH_PRIVATE_KEY']
+    puts 'Writing ssh_private_key into ~/.ssh/'
+    unless ENV['SSH_PRIVATE_KEY']
+      puts 'Aborting because SSH_PRIVATE_KEY is missing'
+      abort
+    end
 
     private_key_path = File.join(SSH_DIR, 'id_rsa')
 
     FileUtils.mkdir_p(SSH_DIR)
     File.write(private_key_path, ENV['SSH_PRIVATE_KEY'])
+    puts 'Updating file permissions on ssh_private_key'
     FileUtils.chmod(0o600, private_key_path)
+    puts 'ssh_private_key successfully written'
   end
 
   # Copied from https://github.com/siassaj/heroku-buildpack-git-deploy-keys/blob/develop/bin/compile#L35
   # with some modifications
   #
   desc 'Write github.com to the known_hosts file on the remote server'
   task :write_known_hosts do
-    abort unless ENV['SSH_PRIVATE_KEY'] && ENV['GITHUB_SSH_RSA']
+    puts 'Writing known_hosts file'
+    unless ENV['SSH_PRIVATE_KEY']
+      puts 'Aborting because SSH_PRIVATE_KEY env var is missing'
+      abort
+    end
+    unless ENV['GITHUB_SSH_RSA']
+      puts 'Aborting because GITHUB_SSH_RSA env var is missing'
+      abort
+    end
 
     known_hosts_path = File.join(SSH_DIR, 'known_hosts')
 
     # Found here: https://github.com/openssh/openssh-portable/blob/0235a5fa67fcac51adb564cba69011a535f86f6b/hostfile.c#L674
     ssh_max_line_length = 8192
     template = %(github.com,%s ssh-rsa #{ENV['GITHUB_SSH_RSA']})
-    ips_per_line = (ssh_max_line_length - template.bytesize) /
-                   '255.255.255.255,'.bytesize
+    ips_per_line = begin
```

Comment:
ruby code linter was struggling with the old syntax since the line ended in a ` / `